test/resources/exchangeRateData/hmrc-exchange-rates-for-2021-monthly.html

```diff
@@ -0,0 +1,1123 @@
+@*
```

Comment:
instead of using this big file, if you have a structure something like 

```
trait ExchangeLinkLoader {
  def getMonthlyUrl(): String
}

class ExchangeLinkLoaderImpl extends ExchangeLinkLoader {
  def getMonthlyUrl(): String = ???
}
```
 it would easier for your unit testing also